### PR TITLE
Nieto/asset profile fe

### PR DESCRIPTION
**Pull request by JMNietoatBusyBee**
"Added List of Asset_profile"
"Consume GET endpoint"

### CODE REVIEW

File: src/app/core/models/asset-profile.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    costCenter: number,
+    economicLife: string,
+    residualLife: string,
+    isActive: string;
+    statusDate: Date;
+    dateCreated: Date,
+    updatedAt?: Date;
+    createdBy: string;
+    updatedBy?:string;
+  }
```
Comment (vernienamca) [inline src/app/core/models/asset-profile.ts:18]: Please correct the code indention.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        <mat-button-toggle-group [formControl]="layoutCtrl" class="mt-4 sm:mt-0">
+          <mat-button-toggle value="boxed">Boxed</mat-button-toggle>
+          <mat-button-toggle value="fullwidth">Full-Width</mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+  </vex-page-layout-header>
+  <vex-page-layout-content [class.container]="layoutCtrl.value === 'boxed'" [class.px-gutter]="layoutCtrl.value === 'fullwidth'" class="-mt-6">
+    <div class="card overflow-auto -mt-16">
+      <div class="bg-app-bar px-6 h-16 border-b sticky left-0 flex items-center" fxLayout="row" fxLayoutGap="10px">
```
Comment (vernienamca) [inline src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.html:18]: Please correct the spacing please.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.container button {
+    border-radius: 5px;
```
Comment (vernienamca) [inline src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.scss:2]: Correct the code indention.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { AssetProfileListComponent } from './asset-profile-list.component';
+
+describe('AioTableComponent', () => {
```
Comment (vernienamca) [inline src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.spec.ts:4]: Name not accurate on the component.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+import { stagger40ms } from '../../../../../@vex/animations/stagger.animation';
+import { UntypedFormControl } from '@angular/forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { PortalService } from 'src/app/core/services/portal.service';
+import { Router } from '@angular/router';
+import { IAssetProfile } from 'src/app/core/models/asset-profile';
+
+@UntilDestroy()
+@Component({
+  selector: 'vex-aio-table',
```
Comment (vernienamca) [inline src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts:21]: Make sure the if you do copy and paste the component. You also change the name of it.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  get visibleColumns() {
+    return this.columns.filter(column => column.visible).map(column => column.property);
+  }
+
+  private _onDestroy$ = new Subject<void>();
+
+  constructor(
+    private _router: Router,
+    private _portalService: PortalService
+  ){}
```
Comment (vernienamca) [inline src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts:74]: Please correct the code spacing please.

File: src/app/core/models/asset-profile.ts (hunk 1 above)
Comment (JMNietoatBusyBee) [reply src/app/core/models/asset-profile.ts:18]: Applied

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.spec.ts (hunk 4 above)
Comment (JMNietoatBusyBee) [reply src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.spec.ts:4]: Applied.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts (hunk 5 above)
Comment (JMNietoatBusyBee) [reply src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts:21]: Noted on this sir, Applied.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts (hunk 6 above)
Comment (JMNietoatBusyBee) [reply src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.ts:74]: Applied.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.html (hunk 2 above)
Comment (JMNietoatBusyBee) [reply src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.html:18]: Applied.

File: src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.scss (hunk 3 above)
Comment (JMNietoatBusyBee) [reply src/app/pages/apps/asset-profile/asset-profile-list/asset-profile-list.component.scss:2]: Applied.

**Review by ferlyncaps — APPROVED** [review]

**Pull request merged**